wiki/Tournaments/NSS/1/en.md

```diff
@@ -0,0 +1,419 @@
+    - Pick, ban, or protect: The referee !rolls the amount of possible maps and uses the result as the player’s choice.
+    - Ready-up timer: !mp start 15 is forced.
+14. For FM maps, mods are optional, but NM = 0.8x, EZ, EZHD, or EZFL = 1.7x, and FL = 1.4x.
+15. If both players somehow get *exactly* the same score, the player with better accuracy wins. If there is still a tie, the map is replayed.
+    - Mods can be changed if a map is replayed for this reason.
+16. In case of a disconnect:
+    - The first time, the map is replayed if the disconnect happened within the first **30 seconds.**
+    - If the same player disconnects again, they lose unless they can prove their score, for example, by Shift+F12 or replay file.
+17. If players do not join the match at the given time, the following penalties apply:
+    - 5 minutes: No warmup
+    - 8 minutes: No protect
+    - 10 minutes: No bans
+    - 15 minutes: Forfeit
+18. If a player becomes silenced during the match:
+    - 5 minutes: Timeouts lost or next pick/ban lost
+    - 10 minutes: Timeouts lost **and** next pick/ban lost
+    - 20 minutes or more: **Forfeit** unless the opponent wants to reschedule
+19. If a player becomes silenced while playing a map, it counts as a disconnect.
+
+### Schedule Rules
+
+1. Qualifiers lobbies should be scheduled at least **6 hours** in advance.
+2. In qualifiers, players may schedule an EX lobby at any time. Depending on referee availability, this may not be granted.
+3. Matches should be scheduled at least **24 hours** in advance.
+4. Reschedule requests should be considered reasonably by the receiving player. If a player appears to be denying reschedule requests unfairly, the administration will intervene.
+5. Reschedule requests with faked confirmation from the other player will result in a forfeit.
+
+### Group Stage
+
+1. During the Group Stage, there are 3 to 5 players in each group, depending on registrations.
+2. These players all play 1 match against each other over two weeks.
+3. The placements are determined by the following criteria, in the order they are applied:
+   - Amount of match wins
+   - Maps won minus maps lost
+   - Direct comparison of a pair's group stage match (e.g. P1 vs P2: 5-1, so P1 gets the better placement)
+
+### Screening
+
+1. Players can be screened for having performed **well above** the level of the rank range or for having suspicious profile stats. The following guidelines (not strict criteria, appealable) are used to determine this:
+   - osu!standard tournament badge(s)
+   - Having a play worth over 300pp
+   - Suspiciously low playtime
+   - Suspiciously recent account registration date
+   - Rank <95k during registration phase
+   - Rank <95k after adding tournament top plays (mixed pp) and unranked osu!lazer plays
+   - Suspicious profile activity
+   - High average scores on pools above 5.5 stars
+   - Consistent high performer in major 100k tournaments (e.g. EQX, 6EC24, GC#1, 6WC23)
```

Comment:
would you mind linking these tournaments? it's very hard to search by acronym in most cases

Reply:
Fixed.